gax-httpjson/src/main/java/com/google/api/gax/httpjson/ApiMessageHttpResponseParser.java

```diff
@@ -86,13 +89,18 @@ public ResponseT read(JsonReader in) {
   }
 
   @Override
-  public ResponseT parse(InputStream httpResponseBody) {
+  public ResponseT parse(InputStream httpResponseBody, Charset httpResponseBodyCharset) {
```

Comment:
Sorry. Didn't realize this was an existing class. I's wrong as currently implemented but we can't change the signature so easily.

Reply:
done